### ISSUE TITLE
Troubles with Installation/which python test

### ISSUE THREAD

**Issue by ZGeoP** [landlab/landlab#454]
Hello all,

Thank you in advance for the help on this. I am fairly new to python and command line so all the help you may be able to give me will be greatly appreciated!...

I am working on installing LandLab, and the Continuum Anaconda distribution (4.3.0 w/ Python 3.6).

When i attempt to run my "which python" test (from the command prompt) after installing and updating all of the necessary addins, I get the following error...

'which' is not recognized as an internal or external command, operable program or batch file.

I have checked my paths, which currently read...

C:\Users\Zach_Phillips\Anaconda3;C:\Users\Zach_Phillips\Anaconda3\Scripts;C:\Users\Zach_Phillips\Anaconda3\Library\bin

Like I said, I'm pretty new to this so this may be an easy fix or I may just be running my which python test from the wrong directory.

Thanks!
Zach

**Comment by mcflugen** [comment]
@ZGeoP: the problem is that you are using Windows but the instructions you are following are for Mac/Linux. Sorry about that - looking at them now I see that there is no indication that they are *not* for Windows. We definitely need to fix that.

Instead of running `which`, just start up Python and see if it says anything about *Continuum Analytics* in the startup message. That should be a good enough test to see that you are running the Python you just installed.

If you're up for it, I would suggest installing *Git BASH*, which will emulate a linux-style command line on Windows (with all of the linux commands, including `which`). You can get it here: https://git-for-windows.github.io. You should then be able to follow along with the instructions. It's totally not necessary, though.

**Comment by ZGeoP** [comment]
Thanks for the speedy response! Will Git BASH help with going through the various tutorials?

**Comment by mcflugen** [comment]
@ZGeoP: No, I don't think it will help with the tutorials. I believe the tutorials are all run from within Python.
